make/langtools/tools/flagsgenerator/FlagsGenerator.java

```diff
@@ -167,8 +168,11 @@ private static Object valueOfValueAttribute(AnnotationMirror am) {
 
     public enum FlagTarget {
```

Comment:
Just flyby comment: symbol kind is an enum, so in _principle_ we could use constants such as MTH, VAR, TYP as targets, if we wanted to connect the decls more with the actual code. But I like your approach as well (as symbol kind has also a lot of other more dubious stuff/noise)